src/components/Body.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Body = ({
+  name, massVal, massExp, volVal, volExp, radius, image, description, moons, type,
+}) => {
+  let moonCount = 0;
+  if (!Array.isArray(moons) || !moons.length) {
+    moonCount = 0;
+  } else {
+    moonCount = moons.length;
+  }
+  return (
+    <div className="col-md-12 single-item">
+      <div className="col-md-4 col-sm-12">
+        <img className="featured-image" src={image} alt={name} />
+      </div>
+      <div className="col-md-8 col-sm-12 description">
+        <h3>{name}</h3>
+        <h5>
+          Type:
+          {' '}
+          {type}
+        </h5>
+        <div className="quick-facts">
+          <p>
+            <strong>Mass: </strong>
+            {massVal}
+            x10
+            <sup>{massExp}</sup>
+            {' '}
+            kg
+          </p>
+          <p>
+            <strong>Volume: </strong>
+            {volVal}
+            x10
+            <sup>{volExp}</sup>
+            {' '}
+            km
+          </p>
+          <p>
+            <strong>Equatorial Radius: </strong>
+            {radius}
+            {' '}
+            km
+          </p>
+          {moonCount !== 0
+        && (
+        <p>
+          <strong>Moons: </strong>
+          {' '}
```

Comment:
- [ ] Great job for adding the details page here but I think it would be great if you make your details page like the given wireframe since it is very important to stick with the given wireframe and it is one of the requirements I will show here what I am talking about 😊♻🔥💯✅

- Hint: you can add a brief from the overview not all the defintion 🙏🏻♻
![image](https://user-images.githubusercontent.com/58553711/217969459-75267001-5443-40a6-8482-f685eb5564f6.png)

- Your design
![image](https://user-images.githubusercontent.com/58553711/218926076-5d671d05-761c-416b-95fc-c50954f241ba.png)



- Given wireframe
![image](https://user-images.githubusercontent.com/58553711/217969385-b8478a67-fae1-4bf2-a409-7c2868cab7a1.png)